tests/infrastructure/golden_images/update_boot_source/utils.py

```
@@ -313,3 +314,48 @@ def wait_for_data_source_unchanged_referenced_volume(data_source: DataSource, vo
+            return data_source.source.name != volume_name
+
+    try:
+        for sample in TimeoutSampler(
+            wait_timeout=TIMEOUT_10MIN,
+            sleep=TIMEOUT_5SEC,
+            func=condition,
+        ):
+            if sample:
+                return
+    except TimeoutExpiredError:
+        LOGGER.error(f"DataSource {data_source.name} was not updated, spec: {data_source.instance.spec}")
```

Ruff (0.16.2)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)